package.json

```diff
@@ -27,9 +27,9 @@
     "test:serverside": "yarn build && node src/serverSideTest.js",
     "storybook": "start-storybook -p 9009",
     "storybook:deploy": "storybook-to-ghpages",
-    "build": "webpack --progress",
-    "build:watch": "webpack --watch",
-    "lint": "tsc --noEmit && eslint --ext js,jsx,ts,tsx src && stylelint \"src/**/*.{css,scss}\"",
+    "build": "tsc && webpack --progress",
+    "build:watch": "tsc && webpack --watch",
+    "lint": "tsc --noEmit --emitDeclarationOnly false && eslint --ext js,jsx,ts,tsx src && stylelint \"src/**/*.{css,scss}\"",
```

Comment:
`noEmit` cannot be used alongside `emitDeclarationOnly` (as of ts 3.8), which is now in tsconfig. So we have to set it to false here, which is the recommended workaround